test/api/client/client.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/konveyor/tackle2-hub/addon"
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/settings"
+)
+
+var Client *addon.Client
+
+func init() {
+	var err error
+	Client, err = New()
+	if err != nil {
+		panic(fmt.Sprintf("Error: Cannot setup API client: %v.", err.Error()))
+	}
+}
```

Comment:
I think we should leave the instantiation of the client to the consumer.

Reply:
Idea behind this is that since test packages (test/application, test/tag, etc.) provide methods for interaction with Hub API, it should be responsible for the client setup. And having this same code in each resource type test package would be a duplication, it is easier to set it up here directly within the client pkg.